cogs/ncplayz.py

```diff
@@ -17,23 +17,23 @@ def __init__(self, bot):
         )
         userinfo_embed.add_field(
             name="User:",
-            value=f"{member}"
+            value={member}
```

Comment:
Invalid syntax.

Reply:
should've been updated in the newer commit